index.d.ts

```diff
@@ -0,0 +1,7 @@
+export declare function escapeId(val: string | string[], forbidQualified?: boolean): string;
+export declare function escape(val: any, stringifyObjects?: boolean, timeZone?: string): string;
+export declare function arrayToList(array: any[], timeZone?: string): string;
+export declare function format(sql: string, values?: any | any[], stringifyObjects?: boolean, timeZone?: string): string;
+export declare function dateToString(date: Date | string, timeZone?: string): string;

```

Comment:
The `date` arg can be a `number` as well.

Reply:
Okay. I will update.